.github/workflows/services-persist-tag.yml

```diff
@@ -15,7 +15,6 @@ on:
 jobs:
   persist-service-tag:
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master' && (${{ github.actor != 'github-actions[bot]' && github.event_name != 'push' || github.event.pusher.name != 'github-actions[bot]' }})
```

Comment:
Similarly here: this workflow can only be called [by another workflow](https://github.com/Knoblauchpilze/user-service/blob/master/.github/workflows/services-persist-tag.yml#L4): it is up to the caller to determine whether they want to persist the tag or not.